### PR TITLE
DG: Add effort appendix

### PR DESCRIPTION
**Pull request by wr1159**
What have you done in this PR?
* Add low-effort Effort section
Dev Testing Steps / Screenshots
image

### CODE REVIEW

**Review by dylkaw — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1165,3 +1165,32 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisite: The last command to be successfully executed is a command that can be undone (View User Guide for list of commands that can be undone).
     2. Test case: `redo`<br>
        Expected: No command is redone. Error details shown in the result display. Command box remains the same.
+
+## **Appendix C: Effort**
+
+### Challenges
+* Hard-to-fix Bug where UI did not update upon change
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
* Hard-to-fix bug where UI did not update upon data change 
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1165,3 +1165,32 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisite: The last command to be successfully executed is a command that can be undone (View User Guide for list of commands that can be undone).
     2. Test case: `redo`<br>
        Expected: No command is redone. Error details shown in the result display. Command box remains the same.
+
+## **Appendix C: Effort**
+
+### Challenges
+* Hard-to-fix Bug where UI did not update upon change 
+* Implementation of `viewMember` / `viewEvent`
+* Designing of `Enrolment` class
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
* Design of `Enrolment` class
```

**Review by ph-nathan — CHANGES_REQUESTED** [review]
If you were a fruit, you'd be a 'fineapple. Commented some nits.

**Review by ph-nathan — APPROVED** [review]
ok

**Pull request merged**